numpy/lib/function_base.py

```diff
@@ -2290,7 +2290,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
 
     # Determine the normalization
     if w is None:
-        fact = X.shape[1] - ddof
+        fact = np.float64(X.shape[1] - ddof)
```

Comment:
I don't think you need this?

Reply:
I did using divide, not with true_divide.  I've changed that back.